tests/Test.ADAL.NET.Unit/AdalDotNetTests.cs

```diff
@@ -853,6 +853,49 @@ public void SimpleRefreshTokenTest()
         }
 
         [TestMethod]
+        [Description("Test for Force Prompt with PromptBehavior.SelectAccount")]
+        public async Task ForcePromptForSelectAccountPromptBehaviorTestAsync()
+        {
+            MockHelpers.ConfigureMockWebUI(new AuthorizationResult(AuthorizationStatus.Success,
+                    TestConstants.DefaultRedirectUri + "?code=some-code"),
+                // validate that authorizationUri passed to WebUi contains prompt=select_account query parameter
+                new Dictionary<string, string> {{"prompt", "select_account"}});
+
+            HttpMessageHandlerFactory.AddMockHandler(new MockHttpMessageHandler
+            {
+                Method = HttpMethod.Post,
+                ResponseMessage = MockHelpers.CreateSuccessTokenResponseMessage(),
+                PostData = new Dictionary<string, string>
+                {
+                    {"grant_type", "authorization_code"}
+                }
+            });
+
+            var context = new AuthenticationContext(TestConstants.DefaultAuthorityHomeTenant, true);
+            TokenCacheKey key = new TokenCacheKey(TestConstants.DefaultAuthorityHomeTenant,
+                TestConstants.DefaultResource, TestConstants.DefaultClientId, TokenSubjectType.User,
+                TestConstants.DefaultUniqueId, TestConstants.DefaultDisplayableId);
+            context.TokenCache.tokenCacheDictionary[key] = new AuthenticationResultEx
+            {
+                RefreshToken = "some-rt",
+                ResourceInResponse = TestConstants.DefaultResource,
+                Result = new AuthenticationResult("Bearer", "existing-access-token",
+                    DateTimeOffset.UtcNow + TimeSpan.FromMinutes(100))
+            };
+
+            AuthenticationResult result =
+                await
+                    context.AcquireTokenAsync(TestConstants.DefaultResource, TestConstants.DefaultClientId,
+                        TestConstants.DefaultRedirectUri, new PlatformParameters(PromptBehavior.SelectAccount));
+            Assert.IsNotNull(result);
```

Comment:
add a check for access token cache count.

Reply:
done